build.assets/Dockerfile-centos7

```diff
@@ -246,11 +246,11 @@ COPY --from=libpcsclite \
     /usr/local/lib/libpcsclite.a \
     /usr/local/lib/
 
-# Download pre-built CentOS 7 assets with clang needed to build BPF tools.
-RUN cd / && curl -L https://s3.amazonaws.com/clientbuilds.gravitational.io/go/centos7-assets.tar.gz | tar -xz
-
 # Copy libbpf into the final image.
 COPY --from=libbpf /opt/libbpf/usr /usr
 
+# Download pre-built CentOS 7 assets with clang needed to build BPF tools.
+COPY --from=ghcr.io/gravitational/teleport-buildbox-centos7-assets:teleport13 /opt/llvm /opt/llvm
```

Comment:
Any way to future-proof that so it doesn't need to be changed every new major version?

Reply:
I was also thinking about it. Technically we've been using Clang 10 since Teleport 4, so there is really no reason to have any versioning, but at some point, we would need to update it. For that, I still think we need the version here as, at that point, older branches will keep using this build, whereas the newer ones need some new tag.